index.js

```diff
@@ -90,7 +85,17 @@ const resolveResponse = (response, options) => {
     return [];
   }
   const responseClone = cloneDeep(response);
-  walkMutate(responseClone, isLink, (link) => normalizeLink(responseClone, link, options.removeUnresolved));
+  const allIncludes = Object.keys(responseClone.includes || {})
+    .reduce((all, type) => ([...all, ...response.includes[type]]), []);
+
+  const allEntries = [...responseClone.items, ...allIncludes];
+
+  allEntries
+    .forEach((item) => (
+      Object.assign(item, {
```

Comment:
If we're in a new line anyway, please use curly braces & `return`

Reply:
Alright :)